content/handbook/tools-and-tips/_index.md

```
@@ -0,0 +1,978 @@
+以下の場所でユーザー名を更新することを忘れないでください:
+
+- [team page](https://gitlab.com/gitlab-com/www-gitlab-com/blob/master/data/team.yml)
+- マーケティングの一員である場合、[Marketing Handbook](https://gitlab.com/gitlab-com/www-gitlab-com/blob/master/source/handbook/marketing/index.html.md)
+- Workday - プロファイルアクションの 'Additional Data' > 'Edit Effective-Dated Custom Object' の下の GitLab Username
+
+#### ステップ 6: 通知を設定する
+
+チームメンバーは、さまざまな方法で GitLab アクティビティを管理することを選択します。あなたに最も適した方法で通知を設定してください。
+
+1. **To-do リスト:** すべてのチームメンバーは、GitLab アカウント内に [to-do リスト](https://docs.gitlab.com/ee/user/todos.html) を表示します。これにより、さまざまなアクションアイテムを追跡できます。完了したアイテムは done としてマークできます。
+1. **メール:** チームメンバーは、[フィルタを使用](#filters) して Gmail アカウントに表示する通知（あれば）を選択できます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)